executor/src/transactions/holder.rs

```diff
@@ -52,6 +52,7 @@ pub(super) struct HolderInfo {
     data: Vec<u8>,
     current_offset: usize,
     hash: B256,
+    recreate: bool,
```

Comment:
Кажется, что в этом флаге нет смысла. В моем предложении он был нужен для того, чтобы все стейджи уместить в один (`HolderFill`), но ты сделал `TxStage::{Create, Delete}`, что позволяет различать стейджи (и в целом проще для понимания) и этот флаг излишен.

Reply:
We still need this because, if we have a transaction to execute from a recovered holder, we need a way to ensure that when we reach the `Final` stage, we continue with `Delete` and `Create`.